lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -776,6 +796,103 @@
             }
           }
 
+          .xseries-action{
+            .xseries-msg{
+              @include float(left);
+              width: flex-grid(9, 12);
+            }
+
+            p+p{
```

Comment:
similarly here, we should try to use a class based identifier instead of the chained element selector